lib/uploadcare/rails/active_record/has_file.rb

```diff
@@ -13,7 +13,7 @@ def has_uploadcare_file(attribute, options={})
           false
         end
 
-        define_method 'build_file' do
+        define_method "build_#{attribute}_file" do
```

Comment:
We have [another PR](https://github.com/uploadcare/uploadcare-rails/pull/57) which aims to fix this issue. I've described the problem with renaming `#build_file` method is [this comment](https://github.com/uploadcare/uploadcare-rails/pull/57#discussion_r148510116)

Reply:
Oops, I didn't realize this issue was already known.  Sorry about that!  I'll wait for the new major version bump, so you can go ahead and close this issue/PR if you'd like.  Thanks for the response!